docs/content/configuration.md

```diff
@@ -384,10 +384,16 @@ keys:
 
 #### AWS Signature
 
-OPA will authenticate with an [AWS4 HMAC](https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-auth-using-authorization-header.html) signature. Two methods of obtaining the
+OPA will authenticate with an [AWS4 HMAC](https://docs.aws.amazon.com/general/latest/gr/sigv4_signing.html) signature. Three methods of obtaining the
```

Comment:
I guess we could omit the number, write "several"... Just so this doesn't need to be kept in sync.